app/javascript/styles/_tab-navigation.scss

```diff
@@ -0,0 +1,76 @@
+.app-tab-navigation {
+  @include govuk-font(19, $weight: bold);
+  @include govuk-responsive-margin(6, "bottom");
+}
+
+.app-tab-navigation__list {
+  @include govuk-clearfix;
+  // left: govuk-spacing(-3);
```

Comment:
@paulrobertlloyd not sure what this is supposed to do exactly but seems to error on negative values. Any ideas?

Reply:
What’s erroring, Sass? Are we using the latest version of `govuk-frontend` – `3.14.0` added support for negative values.

Reply:
Ah no, we're on 3.13.0, will see if I can bump